swift-tests/LottieTests.swift

```diff
@@ -67,6 +67,15 @@ final class LottieTests: XCTestCase {
         XCTAssertTrue(bufferHasContent, "Buffer should have non-zero values after rendering.")
     }
 
+    func testRender_WithAllAvailableFramesOfLottie_Succeeds() throws {
+        let lottie = try Lottie(path: testLottieUrl.path)
+        var buffer = [UInt32](repeating: 0, count: Int(testSize.width * testSize.height))
+
+        for index in 0 ..< lottie.numberOfFrames {
```

Comment:
> nit pick

I assume you tried to test and ensure all frames will not throw errors at line 75.

I am thinking of making it more obvious what we are testing, maybe having `do catch fail` is better?
```swift
do {
    try render
} catch {
    XCFail() 
}
```